homeassistant/components/flickelectric/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "domain": "flickelectric",
```

Comment:
Our standard is to separate words with underscore. Should we rename `flickelectric` to `flick_electric`?